pysmt/walkers/__init__.py

```diff
@@ -33,7 +33,7 @@
 
 """
 
-__all__ = ["DagWalker", "TreeWalker", "IdentityDagWalker"]
+__all__ = ["DagWalker", "TreeWalker", "IdentityDagWalker", "handles"]
```

Comment:
Is this needed? All these names are defined in this `__init__.py` so they should be already importable with `from ... import *` (https://docs.python.org/2/tutorial/modules.html#importing-from-a-package)